### DISCUSSION TITLE
Update Statistics in IndexOptimize

### DISCUSSION THREAD

**Discussion by Nehamnsu** [olahallengren/sql-server-maintenance-solution#446] [Questions & Answers] [answered]
Does the @UpdateStatistics parameter in IndexOptimize update stats with full scan or based on some sampling% on a table?

When i checked the CommandLog table for the commands that were run, I didn't see any specification "with Full Scan"

**Comment by olahallengren** [comment, marked as answer]
@Nehamnsu, by default it is updating the statistics with a default sample.

You can use the parameter ```@StatisticsSample```, if you want to use another sample.
https://ola.hallengren.com/sql-server-index-and-statistics-maintenance.html#StatisticsSample

```@StatisticsSample = 100``` is equal to FULLSCAN.
